packages/image_picker/image_picker_ios/lib/image_picker_ios.dart

```diff
@@ -175,6 +175,47 @@ class ImagePickerIOS extends ImagePickerPlatform {
     );
   }
 
+  Future<List<dynamic>> _getMediaPaths({
```

Comment:
As with Android, you can inline this, and use a helper to do image options class conversion